src/copaw/app/channels/telegram/format_converter.py

```
@@ -0,0 +1,383 @@
+                        f"<b>{cell}</b>" for cell in cells
+                    )
+                else:
+                    # 单列情况
+                    converted_lines[-1] = f"<b>{prev}</b>"
+        table_header_pending = False
+
+    for line in lines:
+        # 1. 代码块检测（检测行中任意位置的 ```）
+        if "```" in line:
+            if in_code_block:
+                # 在代码块内，遇到 ``` 表示结束
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)